### PR TITLE
MEN-2454 Wrappers for mongo

### PR DESCRIPTION
**Pull request by merlin-northern**
Changelog: title
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

File: mongo/wrapper/helper.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package wrapper
+
+import (
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+func MongoGetCollection(client *mongo.Client, dbName string, collectionName string) mongo.Collection {
+	return client.Database(dbName).Collection(collectionName)
```
Comment (alfrunes) [inline mongo/wrapper/helper.go]: Do we really need this wrapper? IMO, I think the new driver API is cleaner than the open-source driver, and I don't see that it makes the code more compact.

Comment (merlin-northern) [reply mongo/wrapper/helper.go]: me making this PR was asking "do we need this?" :)

Comment (merlin-northern) [reply mongo/wrapper/helper.go]: I wanted to add GetAll here as well (wrapper around Find and cursor.All)

Comment (mchalski) [reply mongo/wrapper/helper.go]: I say let's not complicate this and use the driver natively

**Comment by alfrunes** [comment]
This PR has been stale for some time now, closing...

**Pull request closed without merging**